lib/fsInterface.js

```diff
@@ -12,16 +12,18 @@
  * [<code>mkdir</code>]{@link https://nodejs.org/api/fs.html#fs_fs_mkdir_path_options_callback}
  */
 module.exports = (reader) => {
+	const slash =require("slash");
```

Comment:
Missing space after `=`
```suggestion
	const slash = require("slash");
```